x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorIndexService.java

```diff
@@ -1156,8 +1182,8 @@ private static ConnectorSearchResult hitToConnector(SearchHit searchHit) {
     }
 
     /**
-     * This method determines if any documents in the connector index have the same index name as the one specified,
-     * excluding the document with the given _id if it is provided.
+     * This method determines if any records in the connector index have the same index name as the one specified,
```

Comment:
Should it stay `documents` here, if the next line uses `docs`? Or should both be `records`?